mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -14,34 +15,100 @@
  */
 --%>
 
-<%@ include file="/init.jsp" %>
+<%@ include file="init.jsp" %>
```

Comment:
Don't remove the slash, it's a convention that ensures init.jsp is always in the root of the portlet.